[otelaws] Add DynamoDB and SQS X-Ray specific attributes (#2879)
open-telemetry/opentelemetry-go-contrib: CHANGELOG.md

## [Unreleased]

### Added

> - `instrumentation/github.com/aws/aws-sdk-go-v2/otelaws`: Add semconv `messaging.url` & `messaging.system`  attributes to all supporting SQS operations (#2879).

Reviewer: ```suggestion
- Add the `messaging.url` & `messaging.system` attributes to all appropriate SQS operations in the `go.opentelemetry.io/contrib/instrumentation/github.com/aws/aws-sdk-go-v2/otelaws` package. (#2879)
```

Author: Updated.
